### PR TITLE
version 0.3.0 doc

### PR DESCRIPTION
**Pull request by brmorr**
Adding @ckedwards  to verify api on sas-report-page, as well as a change to the link translator code to work for page links. @joylashley for doc review.

### CODE REVIEW

File: documentation/docs/api/SASReportPageElement.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+id: SASReportPageElement
+title: SASReportPageElement
+---
+
+`SASReportPageElement` is a custom HTML element that renders a report page. This element extends <a target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement">`HTMLElement`</a>.
+
+Check out [the Getting Started page](getting-started.md#create-a-custom-html-tag) to learn how to find
+the correct values for `url`, `reportUri`, and `pageName`.
```
Comment (joylashley) [inline documentation/docs/api/SASReportPageElement.md]: Change this to: 
To find the correct values for 'url', 'reportUri', and 'pageName', see [the Getting Started page](getting-started.md#create-a-custom-html-tag).

Comment (brmorr) [reply documentation/docs/api/SASReportPageElement.md]: 99% of this is just copied from SASReportElement and SASReportObjectElement, including this bit.  So I'll make the change everywhere.

File: documentation/docs/api/SASReportPageElement.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Specify the URL of the SAS Viya server that hosts the report. This is the full context root, including the protocol,
+optional port, and host.
+
+### `reportUri: string`
+
+Specify the report URI.
+
+### `pageName: string`
+
+Specify the name of the page from the report to display. Either `pageName` or `pageIndex` should be used, but not both.
```
Comment (joylashley) [inline documentation/docs/api/SASReportPageElement.md]: Change line 43 to:
Specify the name of the report page that you want to display. Either `pageName` or `pageIndex` can be used, but not both.

File: documentation/docs/api/SASReportPageElement.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+### `pageIndex: number`
+
+Specify the index of the page from the report to display. `0` is the first page. Either `pageName` or `pageIndex` should be used, but not both.
```
Comment (joylashley) [inline documentation/docs/api/SASReportPageElement.md]: Change line 47 to:
Specify the index of the report page that you want to display. `0` is the first page. Either pageName or pageIndex can be used, but not both.

File: documentation/docs/api/SASReportPageElement.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+></sas-report-object>
+```
+
+## Attributes
+
+### `authenticationType: string`
+
+Choose the method to authenticate requests to the SAS Viya server.
+- `'guest'` automatically signs in to the SAS Viya server as the guest user.
+- `'credentials'` uses SAS Logon to establish an authenticated session.
```
Comment (joylashley) [inline documentation/docs/api/SASReportPageElement.md]: We don't typically say "SAS Logon". Are we referencing the standard sign-in window for SAS applications? Or is this talking about something else?

Comment (brmorr) [reply documentation/docs/api/SASReportPageElement.md]: This is not new verbiage, and is used everywhere else in the doc where we discuss credentialed login. I feel like this situation is a bit different than explaining the UI of an application.  Here, we are speaking to a developer, and calling out the SAS Logon service by name. It is really the service that is being referenced, not just the UI of a sign in page.

Comment (joylashley) [reply documentation/docs/api/SASReportPageElement.md]: Oh, OK. Thanks for clarifying that it's the service that you're referencing. Ignore my previous comment.

**Comment by brmorr** [comment]
@ckedwards you good with this?

File: documentation/docs/api-reference.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ When you load the library with a script element, the `vaReportComponents` global
 assets are loaded. The `vaReportComponents.loaded` event is dispatched once it is ready.
 
 ```html
-<script async src="https://unpkg.com/@sassoftware/va-report-components@0.2/dist/umd/va-report-components.js"></script>
+<script async src="https://unpkg.com/@sassoftware/va-report-components@0.3.0/dist/umd/va-report-components.js"></script>
```
Comment (ckedwards) [inline documentation/docs/api-reference.md]: With this setup we would have to update the doc every time we made a minor update. I think we should still use "@0.3", and add a note about pinning to exact version for production.

Comment (brmorr) [reply documentation/docs/api-reference.md]: Check out my latest revision.  I said "When used in production, the version should be pinned to the full `major.minor.patch` semantic version."

How is the wording?

**Pull request merged**